src/sentry/static/sentry/app/components/charts/baseChart.jsx

```diff
@@ -54,9 +54,6 @@ class BaseChart extends React.Component {
     // Chart legend
     legend: SentryTypes.EChartsLegend,
```

Comment:
Let's add the `truncate` option in here

Reply:
How come? I noticed TooltipOptions doesn't include it in there, but I do enforce it in SentryTypes if that makes up for it? Also truncate isn't used anywhere in baseChart.jsx

Reply:
Look at `SentryTypes.EchartsTooltip`